tests/conftest.py

```diff
@@ -132,13 +132,17 @@ def schemas_mock(requests_mock: Mocker, schema_url):
     This allows to run "schema import schema afvalwegingen".
     """
     # `requests_mock` is a fixture from the requests_mock package
-    afvalwegingen_json = HERE / "files" / "afvalwegingen.json"
-    requests_mock.get(
-        f"{schema_url}index.json", json={"afvalwegingen": "afvalwegingen/afvalwegingen"}
-    )
-    with open(afvalwegingen_json, "rb") as fh:
+    AFVALWEGINGEN_JSON = HERE / "files" / "afvalwegingen_sep_table.json"
+    CLUSTERS_JSON = HERE / "files" / "afvalwegingen_clusters-table.json"
+    requests_mock.get(f"{schema_url}index.json", json={"afvalwegingen": "afvalwegingen"})
```

Comment:
Change `conftest.schema_url` to return an `URL` and then this can become:
```python
requests_mock.get(schema_url / "index.json", ....
```

Reply:
good one, will do.